nio_cli/commands/config.py

```diff
@@ -14,22 +15,23 @@ def config_project(name='.'):
     pk_token = input('Enter PK Token (optional): ')
     ws_host = pk_host.replace('pubkeeper', 'websocket')
 
-    pk_host_replace = "s/^PK_HOST:.*/PK_HOST: {}/".format(pk_host)
-    pk_token_replace = "s/^PK_TOKEN:.*/PK_TOKEN: {}/".format(pk_token)
-    ws_host_replace = "s/^WS_HOST:.*/WS_HOST: {}/".format(ws_host)
-
-    if pk_host:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_host_replace, env_location),
-            shell=True)
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(ws_host_replace, env_location),
-            shell=True)
-    if pk_token:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_token_replace, env_location),
-            shell=True)
-
+    nenv = open(env_location, 'r')
+    tmp = tempfile.NamedTemporaryFile(mode='w', delete=False)
+    for line in nenv:
+        if re.search('PK_HOST:', line) and pk_host:
+            tmp.write(re.sub('PK_HOST:.*',
```

Comment:
Similarly, on these subs, we should either write the whole line explicitly or use `^` and `$` to grab everything.

Option 1:
```python
tmp.write('PK_HOST: {}'.format(pk_host))
```

Option 2
```python
tmp.write(re.sub('^PK_HOST:.*$', 'PK_HOST: {}'.format(pk_host), line))
```

It seems like the behavior of each is the same, unless you wanted to start including some comments or leading spaces or something.